packages/repository-modernizer/src/restructure-pom.js

```diff
@@ -148,6 +148,18 @@ var RestructurePoms = {
                 pluginObj,
                 conversionStep
             );
+            if(config.parentPom.path == null){
+                let parentPomarr=[project.parentPom.path,project.parentPom.artifactId,project.parentPom.appTitle];
```

Comment:
`parentPomInfo` would be better? Also please add proper indentations in between array members.
Also an object with named properties would be better.
```
{ path : project.parentPom.path, artifactId : project.parentPom.artifactId, appTitle : project.parentPom.appTitle}
```